res/controllers/common-controller-scripts.js

```diff
@@ -16,35 +14,65 @@ String.prototype.toInt = function() {
 // ----------------- Function overloads ---------------------
 
 // Causes script print() calls to appear in the log file as well
-print = function(string) {
+function print(string) {
```

Comment:
I do not like this function declaration syntax in JS. It obscures that all functions are objects, which can be quite confusing for developers not very familiar with JS. Please revert the changes to this file that are not related to this PR.

Reply:
but it makes much more sense for people coming from other languages? They don't have to consider that functions are objects, they just want to call them and the syntax I introduced is much more familiar when looking at the code from that perspective.

Reply:
It makes sense until they come across methods being called on functions and variables assigned to functions.